### PR TITLE
Ci tests

### PR DESCRIPTION
**Pull request by Limezy**

### CODE REVIEW

**Comment by Limezy** [comment]
!testme

**Comment by Limezy** [comment]
!bullseyetestme

**Comment by Limezy** [comment]
!testme
!bullseyetestme
!gogogadgetotitus

File: manifest.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "en": "AI-Powered Photos App for the Decentralized Web",
         "fr": "Gestion de photos en ligne"
     },
-    "version": "2022.28.05~ynh1",
+    "version": "2022.28.05~ynh2",
```
Comment (Thovi98) [inline manifest.json:9]: When you write the version like that, wouldn’t it be an issue if we have to update for `2022.17.06` for example ? Will Yunohost correctly interpret that `2022.17.06` is an higher version than `2022.28.05 ` ?

Comment (Limezy) [reply manifest.json:9]: You are right ! 
I have followed the naming of the upstream app but I may have to change.

**Comment by Limezy** [comment]
!bullseyetestme

**Comment by Limezy** [comment]
I will merge without waiting for Buster tests

**Pull request merged**